### DISCUSSION TITLE
Scaling

### DISCUSSION THREAD

**Discussion by Leohanhart** [agronholm/apscheduler#853] [Q&A]
Dear reader, 

Im trying to find a solution for the following problem.
I would like to use apscedular for a large data-analyses application. I would like to start the application/scedular and then automatic scale workers depending on the load (wich can be big) I can't find a lot of information how to esstablisch this, is this load-balancing done automatticly? 

please tell me with an example how this is done with a redis database. Or anything else what is needed. Thankyou

**Comment by Jdsleppy** [comment]
I'm going to guess you'll have to solve this issue completely outside of APScheduler.  There are too many different deployment environments and paradigms to support them all here.  If you're using an autoscaling group in AWS vs a Kubernetes horizontal pod autoscaler vs whatever, it's going to look very different.
